subxt/src/events/events_type.rs

```diff
@@ -352,6 +213,49 @@ impl<'a, T: Config, Evs: Decode> Events<'a, T, Evs> {
         })
     }
 
+    /// Iterate over all of the events, using metadata to dynamically
+    /// decode them as we go, and returning the raw bytes and other associated
+    /// details. If an error occurs, all subsequent iterations return `None`.
+    ///
+    /// This method is safe to use even if you do not statically know about
+    /// all of the possible events; it splits events up using the metadata
+    /// obtained at runtime, which does.
+    ///
+    /// Unlike [`Events::iter_raw()`] this consumes `self`, which can be useful
+    /// if you need to store the iterator somewhere and avoid lifetime issues.
+    pub fn into_iter_raw(
```

Comment:
Added to support `filter_events`; we need to consume `self` so that we can work with it in our FilterEvents Stream impl.